### PR TITLE
♻️ Maintenance: common redis client in service library

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
This PR brings common code for a Redis client to be used in all services in osparc-simcore repository.

###  Highlights on updated libraries (only updated libraries are included)

- #packages before: 3
- #packages after : 2

|#|name|before|after|upgrade|count|packages|
|-|----|------|-----|-------|-----|--------|
|   1 | deprecated                | 1.2.13          | 🗑️ removed |  | 3 | director-v2⬆️</br>web⬆️🧪 |
|   2 | redis                     | 4.3.4, 4.3.1    | 4.4.0      | *minor*    | 3 | director-v2⬆️</br>web⬆️🧪 |
|   3 | wrapt                     | 1.14.1          | 🗑️ removed |  | 2 | director-v2⬆️</br>web⬆️ |

*Legend*: 
- ⬆️ base dependency (only services because packages are floating)
- 🧪 test dependency
- 🔧 tool dependency

## Related issue/s
Bonus: fixes https://github.com/ITISFoundation/osparc-simcore/issues/3546

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
pair reviewed.

Do not forget to update downstream requirements so they also include the new redis

SEE requirements/how-to-prune-requirements.md

File: packages/service-library/tests/test_redis.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    client = redis_client()
+    assert await client.ping() is True
+    # now let's put down the rabbit service
+    for rabbit_docker_service in (
+        docker_service
+        for docker_service in docker_client.services.list()
+        if "redis" in docker_service.name  # type: ignore
+    ):
+        rabbit_docker_service.remove()  # type: ignore
+    await asyncio.sleep(10)  # wait for the client to disconnect
```
Comment (mrnicegyu11) [inline packages/service-library/tests/test_redis.py:219]: Maybe use `_DEFAULT_LOCK_TTL` here, which is 10sec

**Review by mrnicegyu11 — APPROVED** [review]
thanks!

**Review by GitHK — APPROVED** [review]
Very nice!

**Pull request merged**